modules/quickstart/pages/quick_start.adoc

```diff
@@ -100,90 +107,85 @@ You should receive a token in the response.
 {"authToken":"{auth-token}"}
 ----
 
-Store the authorization token in an environment variable to make it easy to use with `cURL`.
+Store the auth token in an environment variable to make it easy to use with `cURL`.
 
 [source,bash]
 ----
 export AUTH_TOKEN={auth-token}
 ----
 
-Now you can use the REST API.
-
-[source,bash]
-----
-curl --location --request GET 'localhost:8082/v2/keyspaces' \
-  --header "X-Cassandra-Token: $AUTH_TOKEN" \
-  --header 'accept: application/json' \
-  --header 'content-type: application/json'
-----
+Now you're ready to use the REST API for CRUD operations.
 
 === Creating a keyspace using the REST API
 
-Send a `POST` request to `/v2/keyspaces`, setting `name` to the name of the keyspace, and `replicas` to the Cassandra replication factor.
+Send a `POST` request to `/v2/schemas/keyspaces`.
+In this example we use `users_keyspace` for the `name`,
+and `1` for the number of data `replicas`.
 
 [source,bash]
 ----
 curl --location --request POST 'localhost:8082/schemas/keyspaces' \
-  --header "X-Cassandra-Token: $AUTH_TOKEN" \
-  --header 'Content-Type: application/json' \
-  --data-raw '{
+--header 'X-Cassandra-Token: $AUTH_TOKEN' \
```

Comment:
To substitute the AUTH_TOKEN env variable, this line only needs double quotes. Using single quotes will interpret it literally.
```suggestion
--header "X-Cassandra-Token: $AUTH_TOKEN" \
```